kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -85,6 +85,11 @@ logHomeOnPV: false
 # Otherwise, it is ignored.
 logHome: /shared/logs/domain1
 
+# The in-pod location for data storage of default and custom file stores.
+# If not specified or the value is either not set or empty (e.g. dataHome: "") then the
+# configured 'Directory' value in the config.xml, or the default mbean value, will be used.
+#dataHome: /shared/data/domain1
```

Comment:
I like you're "If not specified" wording here better than what I suggested above for the other locations that dataHome has documentation.

Reply:
confirmed resolvved